internal/runners/config/set_test.go

```diff
@@ -15,7 +16,7 @@ func TestSetUnknownKey(t *testing.T) {
 	cfg.Set("unknown", nil)
 
 	outputer := outputhelper.NewCatcher()
-	set := Set{outputer, cfg, nil}
+	set := Set{outputer, cfg, nil, blackhole.New()}
```

Comment:
I would imagine we want some sort of test that verifies the correct analytics "Set" and "Unset" events are emitted, right? Or is it out of scope?